CMakeLists.txt

```diff
@@ -65,6 +65,7 @@ option(WITH_HARFBUZZFONT "Build HarfBuzzFont plugin" OFF)
 option(WITH_ICOIMPORTER "Build IcoImporter plugin" OFF)
 option(WITH_JPEGIMAGECONVERTER "Build JpegImageConverter plugin" OFF)
 option(WITH_JPEGIMPORTER "Build JpegImporter plugin" OFF)
+option(WITH_KTXIMPORTER "Build KtxImporter plugin" OFF)
```

Comment:
I'd vote to keep it as is, but clearly mention it and maybe try to detect KTX1 and spit out a useful error message (as opposed to "wrong file signature"). Bgfx (or rather bimg) [supports version 1.1](https://github.com/bkaradzic/bimg/blob/8355d36befc90c1db82fca8e54f38bfb7eeb3530/src/image.cpp#L3951), but I have absolutely no intuition about usage in the wild. I could check out a few other texture convertors to see if KTX1 is prevalent enough to warrant a name change.

Reply:
I did some quick googling:
- AMD's [Compressonator](https://gpuopen.com/compressonator/) supports KTX 1 and 2
- NVIDIA's [Texture Tools Exporter](https://developer.nvidia.com/nvidia-texture-tools-exporter) supports KTX 1 and 2
- [PVRTexTool](https://developer.imaginationtech.com/pvrtextool/) supports KTX 1 and 2

The others I found don't support KTX2, but are mostly all unmaintained:
- Intel's [ISPC Texture Compressor](https://github.com/GameTechDev/ISPCTextureCompressor) doesn't support KTX at all
- ARM's [astcenc](https://github.com/ARM-software/astc-encoder) supports KTX 1.1
- [etcpack](https://github.com/Ericsson/ETCPACK) only supports KTX 1.1 but it hasn't seen any meaningful updates since 2016
- Google's [etc2comp](https://github.com/google/etc2comp) only supports KTX1 but same thing, no updates for several years
- [etcpak](https://github.com/wolfpld/etcpak) only supports KTX 1.1
- [betsy, used by Godot](https://github.com/darksylinc/betsy) only supports KTX 1.1
- [NVTT](https://github.com/castano/nvidia-texture-tools) only supports KTX 1.1

Reply:
Amazing, thank you. Let's keep it as is, so `KtxImporter` being KTX2 :) 

For the compressors my long-term plan is to wrap them in a way that makes them container-independent (the first one implemented as such is [StbDxtImageConverter](https://doc.magnum.graphics/magnum/classMagnum_1_1Trade_1_1StbDxtImageConverter.html), added yesterday), and thus their limitation to KTX 1 shouldn't be a problem in practice. My main concern was if Blender or Photoshop would support only KTX1 with people *heavily* using that format, but that's fortunately not the case.